.github/workflows/test.yml

```diff
@@ -20,8 +20,9 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 18.0.0
-          - 19
+          - 18.17.0
+          - 20.6.0
```

Comment:
since we now have two minimum versions with the or-ed supported ranges, this prevents accidentally breaking those lower versions